android/build.gradle

```diff
@@ -57,7 +57,7 @@ dependencies {
     //noinspection GradleDynamicVersion
     implementation "com.facebook.react:react-native:+" // From node_modules
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.7.3"
-    implementation "id.vouched.android:vouched-sdk:1.3.7"
+    implementation "id.vouched.android:vouched-sdk:1.3.8"
```

Comment:
I think something went wrong in the rebase I did on your branch in the previous PR, and it went back to version 1.3.7